CHANGELOG

```diff
@@ -3,6 +3,21 @@ https://github.com/rtrlib/rtrlib/commits/master
 
 Release History:
 
+* Version 0.8.0
+    - fix blocking when stopping sockets in some cases
+    - improve error output in TCP transport
+    - properly initialise structures in rtrclient
+    - add callback to allow user created sockets
+    - Fix several compiler warnings
+    - Apply kernel coding style in all source files
+    - Add ability to modify the connect timeout
+    - Improve libssh detection
+    - Fix log prefixes in ssh transport
```

Comment:
... log of prefixes ...

Reply:
It is not about logging of prefixes but about the prefix in the log message indicating the rtrlib module or function the message originates from.

Reply:
Extend log messages to include prefixes when using ssh transport.